test-clients/src/main/java/com/hedera/services/bdd/spec/assertions/ContractFnResultAsserts.java

```diff
@@ -101,6 +103,17 @@ public ContractFnResultAsserts gasUsed(long gasUsed) {
 		return this;
 	}
 
+	public ContractFnResultAsserts contractCallResult(FunctionResult contractCallResult) {
+		registerProvider((spec, o) -> {
+			ContractFunctionResult result = (ContractFunctionResult) o;
+			Assertions.assertArrayEquals(
+					ByteString.copyFrom(contractCallResult.getBytes().toArray()).toByteArray(),
+					result.getContractCallResult().toByteArray(),
+					"Wrong contract call result!");
```

Comment:
Here we can do `Assertions.equals()` and simply compare the ByteString objects, without parsing them to `toByteArray()`